Mathlib/MeasureTheory/Integral/Marginal.lean

```diff
@@ -101,7 +101,14 @@ theorem _root_.Measurable.lmarginal (hf : Measurable f) : Measurable (∫⋯∫
   apply lintegral_dirac'
   exact Subsingleton.measurable
 
-/-- The marginal distribution is independent of the variables in `s`. -/
+/-- The marginal distribution is independent of the variables in `s`.
+
+TODO: This fails with:
+
+elaboration function for 'Std.ExtendedBinder.«termSatisfies_binder_pred%__»'
+  has not been implemented
+  satisfies_binder_pred% i ∉ s
+
```

Comment:
This is the last remaining breakage. Help appreciated.

Reply:
Uggh. Feel free to revert leanprover/std4#425

Reply:
The offending std4 PR is https://github.com/leanprover/std4/pull/425. Maybe @Ruben-VandeVelde knows why this broke.

Reply:
The revert has been committed to std4. This issues has been resolved.